### PR TITLE
Fix allignment issue on the hero section

### PR DESCRIPTION
**Pull request by Apra487**
Fix the allignment issue on the hero section and minor bug fixes

Before 

allignment problem before

After

after

### CODE REVIEW

File: _sass/hero.scss [hunk 1]
```diff
@@ -19,6 +18,8 @@
 
 	h1, a {
```
Comment (sachsom95) [inline _sass/hero.scss:19]: giving global css property means all will be affected is this intentional

File: _sass/hero.scss [hunk 2]
```diff
@@ -42,10 +43,9 @@
 
 	a {
```
Comment (sachsom95) [inline _sass/hero.scss:44]: this seems to be another global css property

Comment (Apra487) [reply _sass/hero.scss:44]: hey h1, a tags are under the hero section.

File: _sass/hero.scss (hunk 1 above)
Comment (sachsom95) [reply _sass/hero.scss:19]: sorry my bad I just looked at the change line only

**Review by sachsom95 — APPROVED** [review]
good to go

**Pull request merged**